astro.config.mjs

```diff
@@ -197,7 +198,18 @@ export default defineConfig({
               { label: 'Overview', link: '/dropins/cart/' },
               { label: 'Installation', link: '/dropins/cart/cart-installation/' },
               { label: 'Styles', link: '/dropins/cart/cart-styles/' },
-              { label: 'Containers', link: '/dropins/cart/cart-containers/' },
+              { label: 'Containers', collapsed: true,
+                items: [
+                  { label: 'Cart', link: '/dropins/cart/containers/cart/' },
+                  { label: 'CartSummaryGrid', link: '/dropins/cart/containers/cart-summary-grid/' },
+                  { label: 'CartSummaryList', link: '/dropins/cart/containers/cart-summary-list/' },
+                  { label: 'Coupons', link: '/dropins/cart/containers/coupons/' },
+                  { label: 'EmptyCart', link: '/dropins/cart/containers/empty-cart/' },
+                  { label: 'EstimateShipping', link: '/dropins/cart/containers/estimate-shipping/' },
+                  { label: 'MiniCart', link: '/dropins/cart/containers/minicart/' },
+                  { label: 'OrderSummary', link: '/dropins/cart/containers/order-summary/' },
+                ]
+              },
               { label: 'Slots', link: '/dropins/cart/cart-slots/' },
```

Comment:
The Lynx team has added a new container: `Order Summary Line`. We may want to add that here.